src/OmniSharp.Abstractions/Models/v1/FileOperation/FileOperationResponse.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
+
+namespace OmniSharp.Models
+{
+    public abstract class FileOperationResponse
+    {
+        public FileOperationResponse(string fileName, FileModificationType type)
+        {
+            FileName = fileName;
+            ModificationType = type;
+        }
+
+        public string FileName { get; }
+
+        [JsonConverter(typeof(StringEnumConverter))]
```

Comment:
at the moment I believe none of the enums we have on the responses are stringified. I think it would be better to stay consistent